src/pint/utils.py

```diff
@@ -1391,3 +1392,38 @@ def FTest(chi2_1, dof_1, chi2_2, dof_2):
             log.warning("Models have equal degrees of freedom, cannot preform F-test.")
         ft = False
     return ft
+
+
+def get_component_type(component):
```

Comment:
Does this return the actual type object? Is it "type" in the python sense you mean here?